packages/flutter/lib/src/material/page.dart

```diff
@@ -36,7 +38,13 @@ class _MountainViewPageTransition extends AnimatedWidget {
     // TODO(ianh): tell the transform to be un-transformed for hit testing
     return new SlideTransition(
       position: listenable,
-      child: child
+      child: new FadeTransition(
+        opacity: new CurvedAnimation(
+          parent: routeAnimation,
+          curve: Curves.easeIn, // Eyeballed from other Material apps.
```

Comment:
Fairly sure. Looking at the native behaviours in slow mo, the opacity comes in really late rather than settling in.